mage/helm/lib.go

```diff
@@ -0,0 +1,43 @@
+// Copyright DataStax, Inc.
+// Please see the included license file for details.
+
+package helm_util
```

Comment:
just noticed the dirs mismatch the package statement - how'd we get in that pattern for mage?

Reply:
yeah we have this in most of our mage dirs. It probably started back with `util` and `sh` dirs being self explanatory when viewing in the filesystem, but when importing as just `sh` and `util` they seemed ambiguous and I think we changed the package names to compensate, but left the dirs more concise.